python/aixexplainer/setup.py

```diff
@@ -33,7 +33,7 @@
     install_requires=[
         "kfserving>=0.5.0",
         "argparse >= 1.4.0",
-        "aix360 >= 0.1.0",
+        "aix360 >= 0.2.0",
```

Comment:
Do we need to change this?

Reply:
It's not necessary for lightgbm server. I upgraded aix360 since the aixexplainer fails to download this library while building docker image in git action.
I will try again with version 0.1.0 after merging the master